packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -26,20 +26,12 @@ export function ProgressBar({
   totalSteps,
   size = "base",
 }: ProgressBarProps) {
-  const percentage = ((currentStep - 1) / (totalSteps - 1)) * 100;
-  const widthPercent = Math.min(100, Math.max(0, percentage));
-  const progressBarContainerClassName = classnames(
-    styles.container,
-    sizes[size],
-  );
+  const percentage = (currentStep / totalSteps) * 100;
   const progressBarContentClassName = classnames(styles.content);
```

Comment:
I am very sure I don't need this, or `styles.container` above, but when I take this one out the progress element loses all of its styling... which I don't understand because I don't even have a classname called `.content` in either the CSS or the rendered HTML anymore?

![image](https://user-images.githubusercontent.com/39704901/100904730-882e5f80-3484-11eb-8070-0049237a4dd6.png)

Reply:
I do still need the `styles[size]` to allow the "small" prop to work